package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "bluebird": "^3.0.1",
+    "jshint": "^2.9.1",
```

Comment:
This should be in `devDependencies` and in addition not required at all. gulp-jshint@^2.0.0 has jshint@2.x in `peerDependencies`. It should be enough.